lib/web/integrations.go

```diff
@@ -504,3 +551,25 @@ func (h *Handler) integrationsMsTeamsAppZipGet(w http.ResponseWriter, r *http.Re
 	}
 	return nil, nil
 }
+
+func (h *Handler) integrationsExportCA(_ http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (interface{}, error) {
+	integrationName := p.ByName("name")
+	if integrationName == "" {
+		return nil, trace.BadParameter("an integration name is required")
+	}
+
+	clt, err := sctx.GetUserClient(r.Context(), site)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	resp, err := clt.IntegrationsClient().ExportIntegrationCertAuthorities(r.Context(), &integrationv1.ExportIntegrationCertAuthoritiesRequest{
+		Integration: integrationName,
```

Comment:
i'm not sure the details of `ExportIntegrationCertAuthorities`, but would this CA be required to update if a user say, update the github client id and secret?

I am planning to allow user to update the id or secret from the `existing integration list` , and it looks like I also need to re-fetch the ca for the user after update

Reply:
no, `ExportIntegrationCertAuthorities` are independent from the OAuth settings. Teleport controls/issues the CAs, and user (GitHub admin) controls/issues the OAuth id/secret. When updating the integration, the CAs do not change.